policy/modules/kernel/kernel.te

```diff
@@ -337,6 +337,7 @@ mls_process_set_level(kernel_t)
 selinux_getattr_fs(kernel_t)
 selinux_load_policy(kernel_t)
 
+
```

Comment:
Unnecessary extra line.

Reply:
Still showing an extra newline.

Reply:
Sorry -- these snuck back in while juggling this PR between multiple workstations.